client/post-editor/editor-revisions-list/header.jsx

```diff
@@ -1,34 +1,37 @@
 /**
- * External dependencies
- *
  * @format
  */
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
 
+/**
+ * External dependencies
+ */
 import React from 'react';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
 
-/**
- * Internal dependencies
- */
-import Button from 'components/button';
-
-const EditorRevisionsListHeader = ( { loadRevision, selectedRevisionId, translate } ) => (
-	<div className="editor-revisions-list__header">
-		<Button
-			compact
-			className="editor-revisions-list__load-revision"
-			disabled={ selectedRevisionId === null }
-			onClick={ loadRevision }
-		>
-			{ translate( 'Load revision in the editor' ) }
-		</Button>
-	</div>
-);
+const EditorRevisionsListHeader = ( { numRevisions, translate } ) => {
+	const classes = classNames( 'editor-revisions-list__header', {
+		'editor-revisions-list__loading-placeholder': ! numRevisions,
```

Comment:
As we require this prop to be a number, I think it would be cleaner to use `numRevisions === 0` and similar condition few lines under. What do you think?

Reply:
I'm indifferent. `! numRevisions` is more concise & `classNames` does a truthiness check anyhow.